client/src/components/InboxUserProfilesGroup/InboxUserProfilesGroup.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./InboxUserProfilesGroup.css";
+import ExtraUsers from "../ExtraUsers/ExtraUsers";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import defaultPic from "assets/images/profile-picture-1.png";
+import defaultPic2 from "assets/images/profile-picture-2.png";
+
+// Function based React Component
+const InboxUserProfilesGroup = props => {
+  // Default Class to apply to Component
+  let classList = `InboxUserProfilesGroup`;
```

Comment:
If there's just the one class, can you put it on line 14 without this variable?

Reply:
The variable classList is also part of the class names on lines 18 and 24. (talking about the corresponding ProfilePicturesGroup.js file since InboxUserProfilesGroup was renamed)